airbytehq/airbyte
# Bump Airbyte version from 0.39.20-alpha to 0.39.21-alpha

*IMPORTANT: Only merge if the platform build is passing!*

Changelog:

de057533fb Reverted changes in SshBastionContainer (#13934)
c283d9d159 Deprecate PART_SIZE_MB in connectors using S3/GCS storage (#13753)
c5783aa799 connectors that published (#13932)
8f602aee96 Source MsSql : incr ver to include changes #13854 (#13887)
bc03fa4e16 13548 Fixed integration tests source-tidb Mac OS (#13927)
174f15d0c0 13546 Fix integration tests source-postgres Mac OS (#13872)
4b116f4889 0.2.5 -> 0.2.6 (#13924)
ec8e5bccf2 Docs: fixed the broken links (#13915)
49d181a198 Normalization: Fix incorrect jinja2 macro `json_extract_array` call (#13894)
b338014b83 🐛 Source Amazon Seller Partner: handle start date for financial stream (#13633)
5852989267 Common code to deserialize a state message in the new format (#13772)
74d16ccb09 Do not hide unexpected errors in the check connection (#13903)
9c1fbea3dd deleting SUMMARY.md since we don't need it for docusaurus builds (#13901)
37b7470139 Edited Amplitude, Mailchimp, and Zendesk Support docs (#13897)
abe3024511 Update airbyte-protocol.md (#13892)
e89846cba9 update api for per stream (#13835)
e208d2c3a2 Display connection state in connection setting page (#13394)
828efb6ddb Update connection update calls to use central utility to ensure connection update has all data (#13564)
72af00a649 octavia-cli: fix workspace not having anonymous_data_collection property (#13869)
e63458fabb Publishing java connectors with better formatted stacktrace in Trace messages (#13852)
e03adaa0a6 Fix md and lg spacing scss vars (#13886)
a2a4ab5553 #13888 source instagram: fix build (#13889)
55aecd92b7 enableReinitialize on account form (#13622)
6ba511cc6d :bug: Source Relational DB : remove unicode nulls from cursors (#13854)
074adefcdc Harshith/test pr 12536 (#13874)
8f709e8638 Source Google Analytics: Add threshold_days option, uncomment statement for incremental tests (#13871)
3bab4c9d1b Persist state to reset connection job config (#13867)
9d060a9053 Add meaningful tests for OTEL client (#13831)
40d58ccd28 Emit orphan running jobs metrics (#13865)
091c5bc976 Add StateType and StateWrapper objects to the model (#13858)
417ba6f696 Fix the running job counter (#13833)
96a4d7515f Updated connector catalog (#13862)
bf52c2afbe 🐛 Server can display streams with the same name in different namespaces (#13855)
1d99b58eb6 Source Amplitude: try-catch BadZipFile exception (#13846)
723bf660d8 Source Hubspot - fix missing data issue (#13837)
b8940ac3d2 source-facebook-marketing: bump to 0.2.53 (#13853)
18b023fbab Tweak schema validation code for readability. (#13721)
62626ee690 🎉 Source Github: use GraphQL for `pull_request_stats` stream (#13763)
ed5d83a44d source-facebook-marketing: 🎉 Add fields to the ads_set stream (#13623)
da95f50555 updated stacktrace format in java trace messages (#13847)
34e2dc36c6 [low-code connectors] Cartesian product stream slicer (#13740)
5f843415db 📚 update MSSQL docs to reflect that we support normalization (#13826)
5d1b902f6d add evan, jonathan, charles as protocol reviews in codeowners (#13776)
4f36bdc541 update readme (#13836)
26f462327b 🎉 Postgres source: prepare for removing strict-encrypt variant (#13823)

Instructions:
- *SQUASH MERGE* this PR - this is necessary to ensure the automated Create Release action is triggered.
- Double check that the Create Release action was triggered and ran successfully on the commit to master (this should only take a few seconds).
- If the Create Release action failed due to a transient issue, retry the action. If it failed due to a non-transient issue, you will need to manually create a release by following these steps:
 1. Pull most recent version of master
 2. Run ./tools/bin/tag_version.sh
 3. Create a GitHub release with the changelog
